### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by blessing-michael**
# Portfolio: details popup window
In this project I; 
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -234,8 +234,95 @@ <h4 class="work-canopy">canopy</h4>
...
+                      ><img src="./images/Ellipse.png" alt="see live"
+                    /></a>
+    
+                </div>
+                    
+                    </div>
+                </div>
+              </div>
+            </div>
+          </div> -->
```
Comment (wayungi) [inline index.html]: - [ ] Please do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history. 🗒️

File: index.js [hunk 2]
```diff
@@ -1,7 +1,160 @@
...
+          /></a>
+            <a href=${search.livelink} class="see-live"
+             ><img src="./images/Enabled.png" class= "pop-enabled" alt="see live"
+          /></a>
+                
+                </div>
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (wayungi) [inline index.js]: - [ ] Good work adding the pop up dynamically as is required. Please make sure the popup is responsive, it fits mobile and desktop sizes properly. This is a project requirement. Currently both desktop & mobile pop ups do not fit properly on the screen. 🥇 
scroll

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @blessing-michael , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] Clicking **see project** pops up the details about the project. 👏🏾 
- [ ] Clicking the close (X) button hides the pop up. 👏🏾 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (blessing-michael) [reply index.js]: Thanks for the review but [wayungi](https://github.com/wayungi) please can you explain further as i do not understand the fit properly on screen. It looks fine on the screen and I don't seem to see the issue you are referring to.  if it's possible add a picture to aid my visual understanding.Thank you

**Review by usorfaitheloho — APPROVED** [review]
Hi @blessing-michael  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**